workspace/errors.go

```diff
@@ -0,0 +1,19 @@
+package workspace
+
+import "errors"
+
+// ErrNotInWorkspace signals that the target directory is outside the configured workspace.
+var ErrNotInWorkspace = errors.New("not in workspace")
+
+// ErrNotExist signals that the target directory could not be located.
+var ErrNotExist = errors.New("not found")
+
+// IsNotInWorkspace checks if this is an ErrNotInWorkspace error.
+func IsNotInWorkspace(err error) bool {
+	return err == ErrNotInWorkspace
```

Comment:
Actually, I deleted the comment because I realized that type assertion is done when creating a struct that matches the error interface. Doing this equality check seems to be ok https://play.golang.org/p/g_C1nEq7Ac